modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/dht/GridCachePartitionsStateValidationTest.java

```diff
@@ -323,4 +330,136 @@ public void waitUntilAllSingleMessagesAreSent() throws IgniteCheckedException {
             allSingleMessagesSent.get();
         }
     }
+
+    /**
```

Comment:
Here and below: There are some redundant comments and commented code left. Please remove all of them.